libnetwork/internal/kvstore/boltdb/boltdb.go

```diff
@@ -15,8 +15,10 @@ import (
 )
 
 var (
-	// ErrBoltBucketOptionMissing is thrown when boltBcuket config option is missing
-	ErrBoltBucketOptionMissing = errors.New("boltBucket config option missing")
+	// ErrBoltPathOptionMissing is returned when New is called with an empty path
+	ErrBoltPathOptionMissing = errors.New("path to the boltdb is missing")
+	// ErrBoltBucketOptionMissing is returned when the bucket name is empty
+	ErrBoltBucketOptionMissing = errors.New("boltdb's bucket name is missing")
```

Comment:
We should remove these errors altogether; they're only used in a single place, and exporting them implies that they're sentinel errors, but they're not used for that.

We can inline the errors, and if needed (not sure if these errors bubble up) change them to be an `errdefs.System` or `errdefs.InvalidParameter`.

It they're not user-configurable, then they should be an `errdefs.System`